lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -147,7 +134,7 @@ define(function(require) {
      * @return {string}  API base url.
      */
     buildApiUrl: function () {
-      return this.apiUrl + "/" + this.nid + "/gallery.json";
+      return this.apiUrl + '?campaigns=' + this.nid + '&status=promoted,approved' + '&exclude=' + this.itemIds + '&count=' + this.requestSize;
```

Comment:
This would be a great case for a template string, imo.

```js
return `${this.apiUrl}?campaigns=${this.nid}&status=promoted,approved&exclude=${this.itemIds}&count${this.requestSize}`;
```